src/Console/Command/FixCommand.php

```diff
@@ -257,6 +261,23 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (null !== $stdErr) {
             $stdErr->writeln(Application::getAboutWithRuntime(true));
 
+            // @TODO remove when parallel runner is mature enough and works as expected
+            if ($resolver->getParallelConfig()->getMaxProcesses() > 1) {
+                $stdErr->writeln(
+                    sprintf(
+                        $stdErr->isDecorated() ? '<bg=yellow;fg=black;>%s</>' : '%s',
+                        'Parallel runner is an experimental feature and may be unstable, use it at your own risk. Feedback highly appreciated!'
+                    )
+                );
+
+                $stdErr->writeln(sprintf(
+                    'Running analysis on %d cores with %d file%s per process.',
```

Comment:
It does not make sense to print it for sequential analysis as there is only one process and files are not grouped in batches. I would leave it as is. Your suggestion is wrong, because we can't reuse `%3$s` as plural form is determined for files (based on files per process), so it has nothing to do with cores 🙂. It's hardcoded `cores` because it's printed only if there's at least 2 CPUs, and IMHO it's fine.

Reply:
> as plural form is determined for files 

ack, we would need to add another param with `$CPUs > 1 ? "s" : ""`

---


> because it's printed only if there's at least 2 CPUs,

That's why I requested
> and write it always (regardless parallel or sequential)

I want to explicitly have this note, and keep it as part of Fixer version, PHP version, whatever else runtime config.